packages/sources/src/components/form/models/form-group-confirm.model.ts

```diff
@@ -0,0 +1,8 @@
+import { Selectable } from '../../selectable/selectable.models';
+
+import { FormGroupBase } from './form-group.base-model';
+
+export interface FormGroupConfirm<TemplateFnReturnType> extends Omit<FormGroupBase<TemplateFnReturnType>, 'label'> {
+  group: 'confirm',
+  selectables: Selectable<TemplateFnReturnType>[];
```

Comment:
Dit moet `selectable: Selectable<TemplateFnReturnType>;` zijn, zie comment in template file voor meer informatie

Reply:
aangepast, maar dat doet het niet werken

Reply:
Zie bovenstaande opmerking met stappenplan.